pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -401,29 +384,24 @@ private BundleData getBundleDataOrDefault(final String bundle) {
                 longTermData.setNumSamples(NUM_LONG_SAMPLES);
             }
         } catch (Exception e) {
-            log.warn("Error when trying to find bundle {} on zookeeper: {}", bundle, e);
+            log.warn("Error when trying to find bundle {} on metadata store: {}", bundle, e);
```

Comment:
should we handle InterruptedException here ?

Reply:
`CompletableFuture.join()` doesn't actually have that in the signature, rather it's already wrapped